config/ember-try.js

```diff
@@ -1,88 +0,0 @@
-'use strict';
```

Comment:
We may eventually want to bring back ember-try if we're going to be sharing this more widely.

Reply:
Ya we'll need to look into it. It's sort of an odd case since we don't actually use anything ember-specific here. We're depending on functionality in ember-cli-deploy. We might need to build a test suite that tests against different versions there. 
Definitely something to consider!